city_scrapers/spiders/chi_ssa_52.py

```diff
@@ -0,0 +1,168 @@
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _replace_meet_chars(self, meet):
+        meet = meet.css("::text").get()
+        meet = meet.replace(".", "")
+        meet = meet.replace("\xa0", "")
+        meet = meet.replace("…", "")
+        meet = meet.replace(",", "")
+        meet = meet.strip()
+
+        return meet
+
+    def _clean_meet(self, meet):
+        """Clean and parse a possible meet <li> item"""
+        payload = {}
+        meet = self._replace_meet_chars(meet)
+        meet = meet.split()
+
+        if len(meet) == 4:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+            }
+
+        if len(meet) == 6:
+            day_week = meet[0]
+            month = meet[1]
+            month_day = meet[2]
+            year = meet[3][:4]
+            time = meet[3][4:]
+            info = " ".join(meet[4:])
+            payload = {
+                "day_week": day_week,
+                "month": month,
+                "month_day": month_day,
+                "year": year,
+                "time": time,
+                "info": info,
+            }
+
+        return payload
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        item = item.replace("\xa0", "").strip()
+        return item
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        months = [
+            "JANUARY",
+            "FEBRUARY",
+            "MARCH",
+            "APRIL",
+            "MAY",
+            "JUNE",
+            "JULY",
+            "AUGUST",
+            "SEPTEMBER",
+            "OCTOBER",
+            "NOVEMBER",
+            "DECEMBER",
+        ]
+
+        try:
+            date = datetime.strptime(
+                f"{item['month_day']} {item['month']}, {item['year']}", "%d %B, %Y"
+            )
+        except ValueError:
+            for month in months:
+                ratio = SequenceMatcher(None, month, item["month"]).ratio()
+                if ratio > 0.5:
```

Comment:
This is an interesting approach! We run into this a lot, so I appreciate this as a standard library tactic we haven't considered before. Is there a reason behind 0.5 as the threshold? Haven't tested, but my only concern would be if they start using abbreviations later on

Reply:
There isn't a concrete reason. Since all i did was testing the possible outcomes with permutations of the months strings. The bigger result i did get was 0.5 comparing `JUNE` with `JUEN`. So I inferred that if the ratio is bigger than 0.5, the word is probably the same. I could be wrong, but it was the only "clever" solution i've found to solve this in an almost automatic way

Reply:
That's a good enough reason! We might want to account for the shorter variations though like "Jun" and "Jul"

Reply:
Ok